src/MapboxStyleParser.ts

```diff
@@ -29,10 +29,26 @@ type SymbolType = {
     iconSymbolizer?: IconSymbolizer;
 };
 
+type OptionsType = {
+    ignoreConversionErrors?: boolean;
+} | undefined;
```

Comment:
` | undefined` should not be needed. Just make it optional where you use it, as you already did.

Reply:
Removed in f8e1e8f, thanks